src/skmultiflow/meta/leverage_bagging.py

```diff
@@ -81,46 +91,48 @@ class LeverageBaggingClassifier(BaseSKMObject, ClassifierMixin, MetaEstimatorMix
 
     References
     ----------
-    .. [1] A. Bifet, G. Holmes, and B. Pfahringer, “Leveraging Bagging for Evolving Data Streams,”
-       in Joint European conference on machine learning and knowledge discovery in databases, 2010,
-       no. 1, pp. 135–150.
+    .. [1] A. Bifet, G. Holmes, and B. Pfahringer, “Leveraging Bagging for
+       Evolving Data Streams,” in Joint European Conference on Machine Learning
+       and Knowledge Discovery in Databases, 2010, no. 1, pp. 135–150.
 
     
     Examples
     --------
     >>> # Imports
-    >>> from skmultiflow.meta import LeverageBaggingClassifier
+    >>> from skmultiflow.meta import LeveragingBaggingClassifier
     >>> from skmultiflow.lazy import KNNClassifier
     >>> from skmultiflow.data import SEAGenerator
     >>> # Setting up the stream
-    >>> stream = SEAGenerator(1, noise_percentage=6.7)
+    >>> stream = SEAGenerator(1, noise_percentage=.067)
     >>> # Setting up the LeverageBagging classifier to work with KNN classifiers
-    >>> clf = LeverageBaggingClassifier(base_estimator=KNNClassifier(n_neighbors=8, max_window_size=2000, leaf_size=30),
-    >>>                                 n_estimators=2)
+    >>> clf = LeveragingBaggingClassifier(base_estimator=
+    >>>                                 KNNClassifier(n_neighbors=8,
+    >>>                                               max_window_size=2000,
+    >>>                                               leaf_size=30)
+    >>>                                 , n_estimators=2)
     >>> # Keeping track of sample count and correct prediction count
     >>> sample_count = 0
     >>> corrects = 0
-    >>> # Pre training the classifier with 200 samples
-    >>> X, y = stream.next_sample(200)
-    >>> clf = clf.partial_fit(X, y, classes=stream.target_values)
     >>> for i in range(2000):
-    ...     X, y = stream.next_sample()
-    ...     pred = clf.predict(X)
-    ...     clf = clf.partial_fit(X, y)
-    ...     if pred is not None:
-    ...         if y[0] == pred[0]:
+    >>>     X, y = stream.next_sample()
+    >>>     pred = clf.predict(X)
+    >>>     clf = clf.partial_fit(X, y, classes=stream.target_values)
+    >>>     if pred is not None:
+    >>>         if y[0] == pred[0]:
     ...             corrects += 1
     ...     sample_count += 1
-    >>> 
-    >>> # Displaying the results
-    >>> print(str(sample_count) + ' samples analyzed.')
+    ... # Displaying the results
+    ... print(str(sample_count) + ' samples analyzed.')
     2000 samples analyzed.
-    >>> print('LeverageBaggingClassifier performance: ' + str(corrects / sample_count))
-    LeverageBagging classifier performance: 0.945
+    >>> print('LeveragingBaggingClassifier performance: ' + str(corrects / sample_count))
+    LeveragingBagging classifier performance: 0.843
```

Comment:
Did the expected performance for this example change because of the modification on `SEAGenerator`'s setup?

Reply:
Looking into the file's history, it seems that the value corresponded to an older version without the `random_state` option.

Reply:
Additionally, the SEA generator now takes values in [0-1] range thus we needed to update the example.

Reply:
Got it! Marking this question as solved. Thank you for checking it out.